packages/fxa-content-server/server/lib/amplitude.js

```diff
@@ -101,6 +101,115 @@ const EVENTS = {
     event: 'signup_code_submit',
   },
 
+  // Add recovery key metrics
```

Comment:
There are no specific test cases for these because the test cases are generic for amplitudeEvents.